Makefile

```diff
@@ -7,12 +7,9 @@
 # This will configure luarocks using 5.1 to use luajit-openresty.
 LUA_VERSION=5.1
 
-LUA_PATH = deps/share/lua/$(LUA_VERSION)/?.lua;deps/share/lua/$(LUA_VERSION)/?/init.lua
-LUA_PATH += ;lua/?.lua;lua/?/init.lua
-LUA_PATH += ;$(shell lua$(LUA_VERSION) -e 'print(package.path)')
-export LUA_PATH
+export LUA_PATH = deps/share/lua/$(LUA_VERSION)/?.lua;deps/share/lua/$(LUA_VERSION)/?/init.lua;lua/?.lua;lua/?/init.lua;$(shell lua$(LUA_VERSION) -e 'print(package.path)')
```

Comment:
It would be nice if you could split this over several lines in some way. Maybe as it was before with the whitespace stripped at the end?